sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.controller.service.ts

```diff
@@ -12,6 +13,20 @@ export class ApplicationOfferingChangeRequestControllerService {
     private readonly applicationOfferingChangeRequestService: ApplicationOfferingChangeRequestService,
   ) {}
 
+  /**
+   * Get the Application Offering Change Request by its id for the ministry.
+   * @param applicationOfferingChangeRequestId the Application Offering Change Request id.
+   * @param options method options:
+   * - `applicationOfferingDetailsReview`: boolean true indicates that only a certain required application offering details are requested.
+   * @returns application offering change request.
+   */
+  async getApplicationOfferingChangeRequest(
+    applicationOfferingChangeRequestId: number,
+    options: {
+      applicationOfferingDetailsReview: boolean;
```

Comment:
Do we really need this option?

Reply:
There are a few properties like `assessedDate` and `assessedBy` that are being returned (only specific to the ministry view). That is why used it to distinguish from the student and institution view.

Reply:
`applicationOfferingDetailsReview` could be something like `hasAdditionalDetails` or `hasAuditAndNoteDetails`

Reply:
Changed it to `hasAuditAndNoteDetails`